apps/backoffice-v2/src/common/components/templates/Providers/Providers.tsx

```diff
@@ -1,12 +1,16 @@
 import { QueryClientProvider } from '@tanstack/react-query';
 import { FunctionComponent, PropsWithChildren } from 'react';
-import { queryClient } from '../../../../lib/react-query/query-client';
-import { AuthProvider } from '../../../../domains/auth/context/AuthProvider/AuthProvider';
+
+import { queryClient } from '@/lib/react-query/query-client';
+import { TooltipProvider } from '../../atoms/Tooltip/Tooltip.Provider';
+import { AuthProvider } from '@/domains/auth/context/AuthProvider/AuthProvider';
 
 export const Providers: FunctionComponent<PropsWithChildren> = ({ children }) => {
   return (
     <QueryClientProvider client={queryClient}>
-      <AuthProvider>{children}</AuthProvider>
+      <AuthProvider>
+        <TooltipProvider>{children}</TooltipProvider>
```

Comment:
TooltipProvider is supposed to wrap each sentence of a Tooltip as far as I know.